src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background: $base-background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__picture {
+    display: block;
+    margin: auto;
+
+    height: 134px;
+  }
+
+  &__name {
+    margin-top: 40px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $base-text-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    margin-top: 4px;
+    display: block;
+
+    color: #616070;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 16px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 24px;
+  }
+
+  &__price-label {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: #616070;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+
+    color: $base-text-color;
+  }
+
+  &__buy-button {
+    text-align: center;
+    margin-top: 16px;
+  }
+
+  &__buy-link {
+    display: block;
+    text-decoration: none;
+    padding: 11px;
+
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    background: $second-theme-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    color: $base-background-color;
+  }
+
+  &:hover &__buy-link {
+    color: $second-theme-color;
+    border-color: $second-theme-color;
+    background: $base-background-color;
```

Comment:
i guess we don't need hover in whole card. Only for button.

Reply:
there wasn't said in the task if button should be hovered or the whole card, but ok, I'll change it